cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetadataStoreDataset.java

```diff
@@ -175,6 +197,48 @@ public boolean exists(MDSKey id) {
     }
   }
 
+  private ImmutablePair<byte[], byte[]> getFuzzyKeyForHACK(MDSKey key) {
```

Comment:
Also file a JIRA to reduce code duplication for generating fuzzy keys to workaround HBase bug